ci/kind/kind-setup.sh

```diff
@@ -233,6 +233,8 @@ function create {
   cat <<EOF > $config_file
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
+featureGates:
+  "NetworkPolicyEndPort": true
```

Comment:
Are the double quotation marks required? If not, maybe remove it to keep same style as L239?

Reply:
Good point. Removed.